Dockerfile.ubi8mini

```diff
@@ -0,0 +1,100 @@
+###############################################################################
```

Comment:
A lot of this Dockerfile is the same as the non-Red Hat one. It seems like it would be feasible to have a base Dockerfile along with a script that could generate the specific Dockefile instances (standard, RedHat, etc.).

Reply:
The intention was to provide a simple way to give compatibility for UBI. A separate Dockerfile seemed the most practical with minimal additional changes to the repo, and followed some existing references that also had multiple Dockerfile (https://github.com/openshift/ansible-service-broker/tree/master/build)

Is acceptance of the PR reliant upon providing the additional changes you described?

Reply:
Hi-
Sorry for the delay. No, they are not required. I am going to try to put something together this weekend, and hopefully get this PR approved and merged on Monday.

Reply:
I added a PR to your fork with some modifications. Please let me know if that works for you. If so, you should be able to approve the PR and the updates should appear here.
Thanks!